k-distribution/tests/regression-new/spec-rule-application/def02-spec.k

```diff
@@ -10,7 +10,7 @@ module DEF02-SPEC
 
     claim <k> mid Y => end ?Z </k>
          <var> _ </var>
-      requires ?Z ==Int incPos(Y)
+      requires Z ==Int incPos(Y)
```

Comment:
`?Z` appears in two locations. You need to update both. Otherwise, you will get two different variable names.